.github/workflows/linux.yml

```diff
@@ -171,6 +171,21 @@ jobs:
         with:
           path: ${{ env.OPENVINO_TOKENIZERS_REPO }}
 
+      - name: Clone Openvino
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          repository: 'openvinotoolkit/openvino'
+          path: ${{ env.OPENVINO_REPO }}
+          ref: ${{ env.OV_BRANCH }}
+          sparse-checkout: |
+            install_build_dependencies.sh
```

Comment:
why is it required?

Reply:
Missed it when resolving conflicts with master branch, fixed